content/docs/rfcs/10/README.md

```diff
@@ -70,38 +111,28 @@ In addition, Waku v2 MAY make use of [libp2p ping protocol](https://docs.libp2p.
 
 for liveness checks between peers, or to keep peer-to-peer connections alive.
 
-For the actual content being passed around, see the [7/WAKU-DATA](/spec/7) spec.
+For the actual content being passed around, see the [14/WAKU2-MESSAGE](/spec/14) and [26/WAKU2-PAYLOAD](/spec/26).
```

Comment:
if I were new to specs then "the actual content being passed around," would sound a bit confusing to me. 
How do we define what is the actual content? also, what do we mean by "being passed around", e.g., in store protocol, we pass history queries, and in response, we get the list of waku messages, in this case, what is the actual content that is being passed?

Reply:
Good point, this should be clarified

Reply:
decided to simplify and remove this, already mentioned elsewhere